### PR TITLE
Add turn down symbol to Optimism L1 deposit transactions in list

### LINKED ISSUE #1851: Signal "deposited txs" on search results
Like we did for blob txs, on optimism superchains there is a "deposited transaction" (type 126): https://specs.optimism.io/protocol/deposits.html#kinds-of-deposited-transactions

it would be nice if we signal those in the search results.

I'd like to suggest this icon for it: https://fontawesome.com/icons/turn-down?f=classic&s=solid

image

that tx type is used for L1->L2 forced txs but also for a system tx which occurs at position 0 in every block, so it would be easy to test.

### PR DESCRIPTION
**Pull request by sealer3**
Closes #1851 

Example:
image

### CODE REVIEW

File: src/components/TransactionLink.tsx [hunk 1]
```diff
@@ -11,9 +12,15 @@ type TransactionLinkProps = {
...
   blob?: boolean;
+  deposit?: boolean;
 };
 
-const TransactionLink: FC<TransactionLinkProps> = ({ txHash, fail, blob }) => (
+const TransactionLink: FC<TransactionLinkProps> = ({
+  txHash,
+  fail,
+  blob,
+  deposit = false,
```
Comment (wmitsuda) [inline src/components/TransactionLink.tsx:22]: I think it is unnecessary to explicitly define default as `false`, no?

**Review by wmitsuda — CHANGES_REQUESTED** [review]

File: src/components/TransactionLink.tsx [hunk 2]
```diff
@@ -25,6 +32,11 @@ const TransactionLink: FC<TransactionLinkProps> = ({ txHash, fail, blob }) => (
         <FontAwesomeIcon icon={faSplotch} />
       </span>
     )}
+    {deposit && (
+      <span className="text-gray-600" title="Deposit transaction">
```
Comment (wmitsuda) [inline src/components/TransactionLink.tsx:36]: I did test some colors on top of your PR, I think `text-emerald-600` looks better, wdyt?

the gray/black-sh color looks too strong and draw too much attention, IMHO.

image

**Review by wmitsuda — COMMENTED** [review]
BTW, could you add a storybook story for the deposit case?

File: src/components/TransactionLink.tsx (hunk 1 above)
Comment (sealer3) [reply src/components/TransactionLink.tsx:22]: Sure, I suppose it's better to be explicit.

**Review by wmitsuda — APPROVED** [review]

**Pull request merged**